### ISSUE TITLE
Inconsistent behavior of `calculateMinMax()` when using WebWorker

### ISSUE THREAD

**Issue by leesei** [cornerstonejs/cornerstoneWADOImageLoader#199]
TL;DR: `calculateMinMax()` is using strict mode when using WebWorker, causing warning `Image smallestPixelValue/largestPixelValue tag is incorrect` and blank content being rendered. #198 

---

In [`processDecodeTask()`](https://github.com/cornerstonejs/cornerstoneWADOImageLoader/blob/v2.1.1/src/imageLoader/decodeImageFrame.js#L15), `strict` has a default value of `false` set in default options.
Yet this is only applied to `calculateMinMax()` when *`useWebWorkers` is false.*

When using WebWorkers, [`handler()`](https://github.com/cornerstonejs/cornerstoneWADOImageLoader/blob/v2.1.1/src/webWorker/decodeTask/decodeTask.js#L54) initialize `strict` from `decodeConfig.decodeTask`, which is `undefined` if we follow the examples. This causes `calculateMinMax()` to run in strict mode.

**Comment by leesei** [comment]
Temporary fix is to add `strict: false` to `decodeConfig.decodeTask`.

**Comment by dannyrb** [comment]
It looks like this has been resolved: https://github.com/cornerstonejs/cornerstoneWADOImageLoader/pull/200

If you're still experiencing this issue, please feel free to open a new ticket and reference this one.
